fv3core/stencils/dyn_core.py

```diff
@@ -28,7 +28,7 @@
 import fv3core.utils.gt4py_utils as utils
 import fv3gfs.util
 import fv3gfs.util as fv3util
-from fv3core.decorators import FixedOriginStencil
+from fv3core.decorators import FrozenStencil
```

Comment:
This is a broader comment, not just about these changes: `FrozenStencil` or `FixedOriginStencil` is not a decorator. It should not be here, or it should be changed to a function that returns an opaque object. + @twicki.

Reply:
Agree that classes are not decorators, but I would ask that we make such a change a separate PR.